test/augmentation/test_augmentation.py

```diff
@@ -2551,3 +2552,159 @@ def test_smoke(self, device, dtype):
+                               [0.3430, 0.4760, 0.2627, 0.3024],
+                               [0.0168, 0.1272, 0.2213, 0.3904]],
+
+                              [[0.6977, 0.8000, 0.1610, 0.2823],
+                               [0.6816, 0.9152, 0.3971, 0.8742],
+                               [0.4194, 0.5529, 0.9527, 0.0362],
+                               [0.1852, 0.3734, 0.3051, 0.9320]],
+
+                              [[0.2621, 0.4020, 0.2245, 0.0472],
+                               [0.3101, 1.0000, 1.0000, 1.0000],
+                               [0.7842, 0.3631, 0.8711, 0.0495],
+                               [0.2067, 0.3609, 1.0000, 1.0000]]],
+
+
+                             [[[0.3736, 0.5783, 0.0666, 0.0994],
+                               [0.2314, 0.4774, 0.3690, 0.6749],
+                                 [0.3430, 0.4760, 0.2627, 0.3024],
+                                 [0.0168, 0.1272, 0.2213, 0.3904]],
+
+                              [[0.6977, 0.8000, 0.1610, 0.2823],
+                                 [0.6816, 0.9152, 0.3971, 0.8742],
+                                 [0.4194, 0.5529, 0.9527, 0.0362],
+                                 [0.1852, 0.3734, 0.3051, 0.9320]],
+
+                              [[0.2621, 0.4020, 0.2245, 0.0472],
+                                 [0.3101, 1.0000, 1.0000, 1.0000],
+                                 [0.7842, 0.3631, 0.8711, 0.0495],
+                                 [0.2067, 0.3609, 1.0000, 1.0000]]]],
+                            device=device,
+                            dtype=dtype)
+
+    def _get_input(self, device, dtype):
+        return torch.tensor([[[[0.4963, 0.7682, 0.0885, 0.1320],
+                               [0.3074, 0.6341, 0.4901, 0.8964],
+                               [0.4556, 0.6323, 0.3489, 0.4017],
+                               [0.0223, 0.1689, 0.2939, 0.5185]],
+
+                              [[0.6977, 0.8000, 0.1610, 0.2823],
+                               [0.6816, 0.9152, 0.3971, 0.8742],
+                               [0.4194, 0.5529, 0.9527, 0.0362],
+                               [0.1852, 0.3734, 0.3051, 0.9320]],
+
+                              [[0.1759, 0.2698, 0.1507, 0.0317],
+                               [0.2081, 0.9298, 0.7231, 0.7423],
+                               [0.5263, 0.2437, 0.5846, 0.0332],
+                               [0.1387, 0.2422, 0.8155, 0.7932]]]],
+                            device=device, dtype=dtype)
+
+    def test_planckian_jitter_blackbody(self, device, dtype):
+        torch.manual_seed(0)
+        f = PlanckianJitter(select_from=1)
+        input = self._get_input(device, dtype)
+        expected = self._get_expected_output_blackbody(device, dtype)
+        assert_close(f(input), expected, atol=1e-4, rtol=1e-5)
+
+    def test_planckian_jitter_cied(self, device, dtype):
+        torch.manual_seed(0)
+        f = PlanckianJitter(mode='CIED', select_from=1)
```

Comment:
```suggestion
        f = RandomPlanckianJitter(mode='CIED', select_from=1)
```
Also seeing  the api now, the current naming can be confusing. Here I would expect that there’s no random applied. Sooner or later we’ll have to provide the non random module at least for exploration. E.g i pas [3,7,4] and just want a batch from this 3 distributions without any random sampling.

Reply:
@edgarriba Few questions about how `PlankianJitter`should behave

- If I have batch of `2` images and I pass `select_from=[20, 4, 5]`, how should it behave?

Reply:
It will randomly sample to sets of indices and apply.That’s why I’m more in favour of the name `RandomPlankianJitter` to avoid confusion.

Reply:
naming `PlanckianJitter` to `RandomPlanckianJitter` perfectly makes sense.

One more question: say `select_from = [10]`, then we do not have randomness in the augmentation applied (meaning we choose the 11th parameter under the pallet params) except the randomness in the certainty of applying (eg. `p = 0.5`, apply aug 50% of the time).

Is this functionality as expected?